.github/workflows/e2e-test.yml

```diff
@@ -6,6 +6,12 @@ on:
       cases:
         description: 'specific cases to be excuted. Sample: ["./aad/a.tests.ts", "./bot/b.tests.ts"]. Set empty to run all cases'
         required: false
+
+      sample-branch:
+        description: "default is 'dev', options: 'v3', 'alpha'"
```

Comment:
now the v3 moved to 'main' branch and 'v3' branch is deprecated, and sample has no branch 'alpha'